components/atom/tooltip/src/index.scss

```diff
@@ -82,7 +88,7 @@ $c-atom-tooltip-colors: (
 }
 
 #{$base-class} {
-  display: block;
+  display: contents;
```

Comment:
👀  The `display: contents` don't have support for IE 11, and has partial support for Safari, Safari iOS, and Samsung Internet, take a look at the [support](https://caniuse.com/css-display-contents)

Reply:
https://github.com/SUI-Components/sui-components/blob/master/components/atom/popover/src/index.scss#L33

2 months ago (nobody has reported any issue)

Reply:
🤷